SonarRunner.Shim/RoslynV1SarifFixer.cs

```diff
@@ -12,15 +12,20 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using System.Diagnostics;
+using System.IO;
+using SonarQube.Common;
 
-namespace SonarQube.Common
+namespace SonarRunner.Shim
 {
-    public static class RoslynV1SarifFixer
+    public class RoslynV1SarifFixer : IRoslynV1SarifFixer
     {
+
+        public static readonly string ReportFilePropertyKey = "sonar.cs.roslyn.reportFilePath";
+
         /// <summary>
         /// Returns true if the given SARIF came from the VS 2015 RTM Roslyn, which does not provide correct output.
         /// </summary>
-        public /* for test */ static bool IsSarifFromRoslynV1(string input)
+        public /* for test */ bool IsSarifFromRoslynV1(string input)
         {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Correct the spelling of 'Sarif' in member name 'RoslynV1SarifFixer.IsSarifFromRoslynV1(string)' or remove it entirely if it represents any sort of Hungarian notation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AIdentifiersShouldBeSpelledCorrectly)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The 'this' parameter (or 'Me' in Visual Basic) of 'RoslynV1SarifFixer.IsSarifFromRoslynV1(string)' is never used. Mark the member as static (or Shared in Visual Basic) or use 'this'/'Me' in the method body or at least one property accessor, if appropriate. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AMarkMembersAsStatic)